frontend/src/views/edit-project/RepresentationArea.tsx

```diff
@@ -36,7 +36,10 @@ export const RepresentationArea = ({
 }) => {
   let content;
   if (!displayedRepresentation) {
-    content = <OnboardArea selection={selection} setSelection={setSelection} />;
+    if (window.performance && performance.navigation.type !== PerformanceNavigation.TYPE_RELOAD) {
```

Comment:
As discussed during the meeting, the issue is a matter of wrong lifecycle in our state machines. Do not try to detect reload events. On top of that you are relying on a [deprecated API](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigation/type)